lib/driver.js

```diff
@@ -46,6 +47,7 @@ const TO_PROXY_ANDROID_ONLY = [
 
 const TO_PROXY_IOS = TO_PROXY_IOS_ONLY.concat(TO_PROXY_COMMON);
 const TO_PROXY_ANDROID = TO_PROXY_ANDROID_ONLY.concat(TO_PROXY_COMMON);
+const TO_PROXY_MAC = TO_PROXY_COMMON;
```

Comment:
Do you think it will be better to have something like this: 

```
const TO_PROXY_MAC_ONLY = [];
const TO_PROXY_MAC = TO_PROXY_MAC_ONLY.concat(TO_PROXY_COMMON);
```
In case we want to add MAC specific commands down the road

Reply:
Hmm looking more into the Mac driver and comparing it to the ios/android driver I am not sure if we are even able to do such thing. All the "base" proxied commands that we concatenated to the platform specific commands are found in both ios and android drivers 
```
const TO_PROXY_COMMON = [
  'background',
  'closeApp',
  'getLog',
  'getLogTypes',
  'getOrientation',
  'getStrings',
  'launchApp',
  'lock',
  'removeApp',
  'setOrientation',
];
``` 
but non of those are found in the mac driver, so I am not sure why we are defining `const TO_PROXY_MAC = TO_PROXY_COMMON;` and when we start the mac session `this.proxyAllowList = TO_PROXY_MAC;`?

Reply:
You are correct. Most, if not all, are not supported by the Mac Driver. I just put it in as we'd be able to support it right away once/if they end up adding any.